pkg/proxy/iptables/proxier.go

```diff
@@ -982,8 +982,8 @@ func (proxier *Proxier) syncProxyRules() {
 		// implemented and the DNS operator is updated to use it.
 		if svcPortNameString == "openshift-dns/dns-default:dns" || svcPortNameString == "openshift-dns/dns-default:dns-tcp" {
 			for _, ep := range clusterEndpoints {
-				if ep.GetIsLocal() {
-					klog.V(4).Infof("Found a local endpoint %q for service %q; preferring the local endpoint and ignoring %d other endpoints", ep.String(), svcPortNameString, len(clusterEndpoints) - 1)
+				if ep.IsLocal() {
+					klog.V(4).Infof("Found a local endpoint %q for service %q; preferring the local endpoint and ignoring %d other endpoints", ep.String(), svcPortNameString, len(clusterEndpoints)-1)
```

Comment:
please squash this fix into the "Prefer local endpoint for cluster DNS service" commit